nest/common/exceptions.py

```diff
@@ -1,3 +1,5 @@
+from http import HTTPStatus
```

Comment:
Please create a md file under docs that demonstrate the usage of the different exceptions, documentation is the most important part of it because this is how people are going to use those pieces of code. After creating the markdown, add it to the static docs site under the mkdosc.yml file

Reply:
done!